# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: target/mnemosyne/pkg_patch/runit-shutdown.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- runit-shutdown-0.2.orig/Makefile	2006-04-08 12:23:33.000000000 +0300
+++ runit-shutdown-0.2/Makefile	2008-07-29 22:35:04.000000000 +0300
@@ -8,9 +8,8 @@
 all:
 	@echo "nothing to do yet"
 install:
-	for f in halt runlevel shutdown killall5; do \
+	for f in halt runlevel shutdown; do \
 		install -m 755 src/$$f.sh $(DESTDIR)$(SBINDIR)/$$f ; \
 	done
 	ln -snf halt $(DESTDIR)$(SBINDIR)/reboot
 	ln -snf halt $(DESTDIR)$(SBINDIR)/poweroff
-	ln -snf $(SBINDIR)/killall5 $(DESTDIR)$(BINDIR)/pidof
